src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -108,6 +113,23 @@ async function runCell(editorGroupsService: IEditorGroupsService, context: INote
 	}
 }
 
+async function ensureNotebookEditorVisible(editorGroupsService: IEditorGroupsService, context: INotebookActionContext, editorService: IEditorService): Promise<void> {
```

Comment:
This is strange to me, why not open the editor instead using the option `revealIfOpened`?

Reply:
100% the right decision, I hadn't seen that parameter. Thanks 👍

Reply:
pushed a commit to include the `revealIfOpened` param to the `openEditor()` call. Not fully sure if that's the full scope of the change you were talking about, sent you a note on slack with a more detailed Q.

Reply:
think I sorted it out thanks to some help from @amunger on finding the right openEditor overload. Lmk if there's something else you have in mind.